docs/pages/server-access/getting-started.mdx

```diff
@@ -0,0 +1,347 @@
+     --auth-server=tele.example.com:443
+   ```
+
+   - Replace the `auth-server` value with the public proxy address of the machine you wish to connect to. By default, the subdomain `tele.example.com` will be available on port `443`.
+   - Supply the secured path to the new Node's `token.file`.
+
+3. Create a user to access the `tele.example.com` Web UI through the following command:
+
+   ```bash
+   # Add a user
+   sudo tctl users add tele-admin --roles=editor,access --logins=root,ubuntu,ec2-user
+   ```  
+
+   This will generate an initial login link where you can set a password and set up Two-Factor Authentication for `tele-admin`.
+
+   <Admonition type="note" title="Note">
+     We've only given `tele-admin` the roles `editor` and `access` according to the *Principle of Least Privilege* (POLP).
+   </Admonition>
+
+4. You should now be able to view both Nodes in the Teleport Web interface after logging in as `tele-admin`:
+
+   <Figure
+     align="center"
+     bordered
+     caption="Both Nodes in the Web UI"
+   >
+     ![Both Nodes in the Web UI](../../img/server-access/teleport_ui.png)
+   </Figure>
+
+## Step 3/4. SSH into the server
+
+Now, that we've got our cluster up and running, let's see how easy it is to connect to our two Nodes. 
+
+We can use `tsh` to SSH into the cluster:
+
+1. On your local machine, log in through `tsh`: 
+
+   ```bash
+   # Log in through tsh
+   sudo tsh login --proxy=tele.example.com:443 --user=tele-admin
+   ```
+   
+   You'll be prompted to supply the password and One Time Passcode we set up previously.
+
+2. `tele-admin` will now see:
+
+   ```bash
+   Profile URL:        https://tele.example.com:443
+     Logged in as:       tele-admin
+     Cluster:            tele.example.com
+     Roles:              access, editor
+     Logins:             root, ubuntu, ec2-user
+     Kubernetes:         disabled
+     Valid until:        2021-04-30 06:39:13 -0500 CDT [valid for 12h0m0s]
+     Extensions:         permit-agent-forwarding, permit-port-forwarding, permit-pty
+   ```
+
+   `tele-admin` is now logged into the `tele.example.com` cluster and *Bastion Host* Node through Teleport SSH.
+
+3. `tele-admin` can now execute the following to find the cluster's `nodenames`. `nodenames` are used for establishing SSH connections:
+
+   ```bash
+   # Display cluster resources
+   sudo tsh ls
+   ```
+
+   The *Bastion Host* Node is located on the bottom line below:
+
+   ```bash                        
+   Node Name        Address        Labels                                 
+   ---------------- -------------- -------------------------------------- 
+   ip-172-31-35-170 ⟵ Tunnel                                              
+   ip-172-31-41-144 127.0.0.1:3022 env=example, hostname=ip-172-31-41-144 
+   ```
+
+4. `tele-admin` can SSH into the *Bastion Host* Node by running the following command locally:
+
+   ```bash
+   # Use tsh to ssh into a Node
+   sudo tsh ssh root@ip-172-31-41-144
```

Comment:
@inertial-frame you don't need `sudo` for `tsh` to work, so you can safely remove it